src/Operators/vorticity_operators.jl

```diff
@@ -14,11 +16,11 @@
 #####
 
 """
-     Γᶠᶠᶜ(i, j, k, grid, u, v)
+    Γᶠᶠᶜ(i, j, k, grid, u, v)
 
- The vertical circulation associated with horizontal velocities ``u`` and ``v``.
- """
-@inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v)
+The vertical circulation associated with horizontal velocities ``u`` and ``v``.
+"""
+@inline function Γᶠᶠᶜ(i, j, k, grid::ConformalCubedSpherePanel, u, v)
```

Comment:
This also means that the panel cannot be used stand-alone, right? It might be good to encode that somewhere, eg prevent people from building single simulations on a conformal cubed sphere panel.

Reply:
Another idea is to actually encode, in `conformal_mapping`, whether the panel is part of a larger cubed sphere or whether it is stand-alone. Then we could use it stand-alone which presumably we want to do for testing (not sure if we actually do that though)

Reply:
Hm... I think you are making a good point here @glwagner.

See https://github.com/CliMA/Oceananigans.jl/pull/3868

Reply:
```Julia
julia> using Oceananigans, Oceananigans.Grids: conformal_cubed_sphere_panel

julia> grid = conformal_cubed_sphere_panel(size=(6, 6, 1), z=(0, 1), topology=(Bounded, Bounded, Bounded))
6×6×1 OrthogonalSphericalShellGrid{Float64, Bounded, Bounded, Bounded} on CPU with 1×1×1 halo and with precomputed metrics
├── centered at: North Pole, (λ, φ) = (0.0, 90.0)
├── longitude: Bounded  extent 90.0 degrees variably spaced with min(Δλ)=6.71593, max(Δλ)=15.5107
├── latitude:  Bounded  extent 90.0 degrees variably spaced with min(Δφ)=6.71593, max(Δφ)=15.5107
└── z:         Bounded  z ∈ [0.0, 1.0]      regularly spaced with Δz=1.0

julia> model = HydrostaticFreeSurfaceModel(; grid)
HydrostaticFreeSurfaceModel{CPU, OrthogonalSphericalShellGrid}(time = 0 seconds, iteration = 0)
├── grid: 6×6×1 OrthogonalSphericalShellGrid{Float64, Bounded, Bounded, Bounded} on CPU with 1×1×1 halo and with precomputed metrics
├── timestepper: QuasiAdamsBashforth2TimeStepper
├── tracers: ()
├── closure: Nothing
├── buoyancy: Nothing
├── free surface: SplitExplicitFreeSurface with gravitational acceleration 9.80665 m s⁻²
│   └── substepping: Barotropic time step equal to 2.044 days
├── advection scheme: 
│   └── momentum: Vector Invariant, Dimension-by-dimension reconstruction
└── coriolis: Nothing

julia> simulation = Simulation(model, Δt=1.0, stop_time=10)
Simulation of HydrostaticFreeSurfaceModel{CPU, OrthogonalSphericalShellGrid}(time = 0 seconds, iteration = 0)
├── Next time step: 1 second
├── Elapsed wall time: 0 seconds
├── Wall time per iteration: NaN days
├── Stop time: 10 seconds
├── Stop iteration : Inf
├── Wall time limit: Inf
├── Callbacks: OrderedDict with 4 entries:
│   ├── stop_time_exceeded => Callback of stop_time_exceeded on IterationInterval(1)
│   ├── stop_iteration_exceeded => Callback of stop_iteration_exceeded on IterationInterval(1)
│   ├── wall_time_limit_exceeded => Callback of wall_time_limit_exceeded on IterationInterval(1)
│   └── nan_checker => Callback of NaNChecker for u on IterationInterval(100)
├── Output writers: OrderedDict with no entries
└── Diagnostics: OrderedDict with no entries

julia> run!(simulation)
[ Info: Initializing simulation...
[ Info:     ... simulation initialization complete (4.523 seconds)
[ Info: Executing initial time step...
[ Info:     ... initial time step complete (9.782 seconds).
[ Info: Simulation is stopping after running for 17.305 seconds.
[ Info: Simulation time 10 seconds equals or exceeds stop time 10 seconds.
```